Extension/src/LanguageServer/configurations.ts

```diff
@@ -1337,8 +1350,39 @@ export class CppProperties {
             envText = curText.substr(envStart, envEnd);
             const envTextStartOffSet: number = envStart + 1;
 
+            // Check if all config names are unique.
```

Comment:
Since `handleSquiggles()` is getting long, could the code that does `// Check if all config names are unique.` be factored out to a different function just for checking config names?

Reply:
Don't need to refactor for insiders. if you have time, you can refactor later